src/build/app/mod.rs

```diff
@@ -988,6 +1016,7 @@ impl<'b> App<'b> {
             ..Arg::default()
         });
         self.args.push(f(a));
+        self.panic_on_missing_help();
```

Comment:
Similar to my above comment, don't think it should be called here.

Reply:
Agreed.